Corona-Warn-App/src/main/java/de/rki/coronawarnapp/appconfig/CoronaTestConfig.kt

```diff
@@ -0,0 +1,22 @@
+package de.rki.coronawarnapp.appconfig
+
+import de.rki.coronawarnapp.appconfig.mapping.ConfigMapper
+
+interface CoronaTestConfig {
+    val coronaRapidAntigenTestParameters: CoronaRapidAntigenTestParametersContainer
+
+    interface Mapper : ConfigMapper<CoronaTestConfig>
+}
+
+data class CoronaRapidAntigenTestParametersContainer(
+    val hoursToDeemTestOutdated: Int = DEFAULT_HOURS
+) {
+    companion object {
+        const val DEFAULT_HOURS = 48
```

Comment:
`DEFAULT_HOURS` could be `private`, or is it intended to be used somewhere else?

Reply:
👍🏼 , will do in another PR , I planed to used publicly, but then I refactored it